ballerina/Ballerina.toml

```diff
@@ -1,17 +1,20 @@
 [package]
 org = "ballerina"
 name = "mqtt"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
We need to bump to the next minor version since we are updating the distribution

Reply:
Bumped. It doesnt matter anyway since we are going with 1.0 in the next release